packages/studio-ui/src/web/components/Layout/BottomPanel/Debugger/index.tsx

```diff
@@ -132,15 +132,19 @@ export class Debugger extends React.Component<Props, State> {
     }
   }
 
-  getEvent = async (eventId: string): Promise<sdk.IO.IncomingEvent> => {
+  getEvent = async (messageId: string): Promise<sdk.IO.IncomingEvent> => {
+    if (!messageId) {
+      return
+    }
+
     const eventsCache = this.state.eventsCache
 
-    const existing = eventsCache.find(x => x.id === eventId)
+    const existing = eventsCache.find(x => x.messageId === messageId)
     if (existing) {
       return existing
     }
 
-    const { data: event } = await axios.get(`${window.BOT_API_PATH}/mod/extensions/events/${eventId}`)
+    const { data: event } = await axios.get(`${window.BOT_API_PATH}/messaging/message-to-event/${messageId}`)
```

Comment:
Does this route really exist ?

Reply:
Yes implemented here https://github.com/botpress/botpress/pull/5183